### DISCUSSION TITLE
Support for Samsung Gear Sport (SM-R600)?

### DISCUSSION THREAD

**Discussion by TrustedKiller** [200Tigersbloxed/HRtoVRChat_OSC#15] [Q&A]
Didn't see any lists that show they support the first Samsung Gear Sport model.  Is it possible to get support for it?

**Comment by 200Tigersbloxed** [comment]
Does the Gear Sport model run WearOS? It should support Stromno if it is WearOS.

**Reply by TrustedKiller** [reply]
It does, but Stromno, according to the list of support devices, only supports Tizen OS versions 5.5 and up.  The first Gear Sport is 4.0.0.7.  Unless I can updated it beyond that, that's the latest my device will automatically update to.

**Reply by 200Tigersbloxed** [reply]
It looks like Stromno has a WearOS app, you could try that, but keep in mind the app is 5 bucks still :(

https://play.google.com/store/apps/details?id=com.stromno.wear&src=home
https://www.stromno.com/

**Reply by TrustedKiller** [reply]
If it works, then worth it.  I'll check it out and report my findings.

**Reply by TrustedKiller** [reply]
So it appears the Gear Sport does not run WearOS, it runs Tizen OS, and the version it runs is not compatible with anything on the list of supported devices.  So Stromno would be a no-go for this.

What would need to be done to get the Gear Sport working with VRChat?  It has apps for Heart Rate monitoring that can be sent to PC for stuff like OBS.  I can't imagine the leap to OSC would be at all difficult.

**Reply by 200Tigersbloxed** [reply]
Which software do you use for OBS? Could you leave a link?

**Reply by TrustedKiller** [reply]
I don't actively use it for OBS, I just know that it can be used for that purpose.  The one I came across is called "HeartRateToWeb."

Watch - Galaxy Store link: https://galaxystore.samsung.com/detail/tUhSWQRbmv?langCd=en
PC - Github link: https://github.com/loic2665/HeartRateToWeb

**Reply by TrustedKiller** [reply]
Okay, after diving into the software a little bit, it writes the BPM to a text file.  Isn't there an option to pull from a text file for VRChat?

**Reply by 200Tigersbloxed** [reply]
Yes, you can use the textfile hrType, and set the textfilelocation config to where the textfile is located. Verify that the textfile only writes a number, otherwise the application will fail to read the number.
